docs/lore/characters/jargon.mdx

```diff
@@ -0,0 +1,386 @@
+
+```
+
+### sean-
+
+```
+A prefix used to indicate that something is provided by or part of the game itself. Common-ish examples: seanscript, seanbug, seanuser
+
+```
+
+### sector
+
+```
+All public and not hidden scripts are assigned to a sector. Sectors can be listed with scripts.fullsec, scripts.highsec, ... To view the scripts in a sector, you must join that sector's channel, e.g. if you want to see the scripts in FORM_ALPHA_0, you must join the chat channel FORM_ALPHA_0. After joining that sector's chat channel, you can view the scripts in it with scripts.<seclevel> {sector:"<that sector>"}
+
+```
+
+### sentience
+
+```
+See PC
+
+```
+
+### shift
+
+```
+The process that assigns a script to a sector. Shifting a script takes 15 minutes, and is required with any sec level change of a public script, or when first making a script public.
+
+```
+
+### stuck NPCs
+
+```
+Some NPCs can get stuck by malfunctioning crackers (typically by calling them after they pop or solving the last lock within the last 1000ms of your script run). Rather than popping, the NPC persists, possibly partially deleted.
+
+```
+
+### subscript
+
+```
+A way of calling a script inside another script. The format is #Xs.script.name(args), where X is the sec level (f, h, m, l, n, or 4, 3, 2, 1, 0).
+
+```
+
+### t0
+
+```
+See uninit
+
+```
+
+### t1 / t2 / t3 / t4
+
+```
+Shorthand for tier 1 through tier 4. Used for upgrade tiers, init tiers, and NPCs.
+
+```
+
+### truthworth
+
+```
+A meme originating from the sripts user. Trustworthy. Sometimes used jokingly
+
+```
+
+### truthy
+
+```
+In JS, any value that, if present in an if() statement, causes the if body to be entered. Everything that isn't falsey is truthy. See also falsey
+
+```
+
+### typosquat
+
+```
+Scripts, usually scams, with very similar names to trust scripts. Users with names similar to trust users (such as kernal, sus, sripts) are typically employed, though things like upgrades.manage are arguably typosquats as well.
+
+```
+
+### uninit
+
+```
+A user that has not yet initialized their system.
+
+```
+
+### opsec
+
+```
+Short for Operational Security, a series of methods involved in hiding information on how you operate.
+
+```
+
+### scropotrable
+
+```
+"scriptorable". meme derived from a typo made by user dtr. http://i.imgur.com/Qd4noFn.png
+
+```
+
+### duck-typed / duck-typing
+
+```
+An informal description of a language, meaning that it (at least usually) doesn't use strong typing but instead relies on, for example, anything that has an x and y property being essentially a 2D point. If it looks like a duck, and quacks like a duck, it must be a duck. JS is (mostly) duck-typed.
+
+```
+
+### usnack/usnax
+
+```
+no hints 4 u, go away stinky -j4c0b
+
+```
+
+### glam
+
+```
+Upgrade type that has no actual function (beyond being shiny). Typically distributed as trophies during events.
+
+```
+
+### Alice number
```

Comment:
do we need this?

Reply:
This was mostly relevant to the discord's risk timestamps, which have been gone for a while now